frontend/src/components/pages/login/Login.tsx

```diff
@@ -26,7 +23,7 @@ const Login: React.FC = () => {
             <div className="google-logo-container">
               <img src={GoogleLogo} alt="google-logo" />
             </div>
-            <Link to='/settings' className="login-btn-link" onClick={saveUserToStorage}>Sign in with google</Link>
+            <Link to='/settings' className="login-btn-link" onClick={() => saveUserToStorage(user)}>Sign in with google</Link>
```

Comment:
not sure we need to navigate to settings during signin. Also where this user comes from? Since we have the BE now - we need to move all mocks from FE to BE